dictionaries/mn_MN/src/hunspell/package.json

```
@@ -30,5 +30,8 @@
     "index.dic",
     "index.js",
     "index.map"
-  ]
+  ],
+  "scripts": {
+    "start": "node download.js"
+  }
```

```suggestion
  ]
```

This isn't needed if the changes to `download.js` are made.